tendermint/src/validator.rs

```diff
@@ -318,7 +319,11 @@ impl ProposerPriority {
     }
 }
 
-/// Updates to the validator set
+/// A change to the validator set.
+///
+/// Used to inform Tendermint of changes to the validator set.
+///
+/// [ABCI documentation](https://docs.tendermint.com/master/spec/abci/abci.html#validatorupdate)
```

Comment:
```suggestion
/// [ABCI documentation](https://github.com/tendermint/tendermint/blob/v0.34.x/spec/abci/abci.md#validatorupdate)
```

We currently do not make the spec available via the docs site, and the one from `master` is no longer relevant. We also keep the specification relevant to the specific Tendermint release series on each release branch, so it's easier to see diffs in the specs between major releases of Tendermint.

Reply:
I think we can probably keep this as-is for this PR, but in a follow-up PR we should update all the spec references to rather point to the `v0.34.x` branch of the Tendermint repo. I've created #1224 to track this.